### PR TITLE
Final model table and benchmarking data

### PR DESCRIPTION
**Pull request by jeromekelleher**
Table looks like this now:

Screenshot from 2019-12-19 17-02-39

I didn't simulate the longest chromosome because all the simulations wouldn't have run this afternoon. All we're showing here is roughly what you might expect in a best case scenario. There's no point in trying to make statements about how long any given simulation might take, since it all depends on the parameters in a complicated way.

The main message we want to get across is, you'll probably be able to run the simulations you're interested in a reasonable amount of time.

### CODE REVIEW

File: popSimManu.tex [hunk 1]
```diff
@@ -302,14 +304,24 @@ \subsection*{The \stdpopsim Catalog}
...
+\makebox[\textwidth][c]{
+    \begin{footnotesize}
+    \input{model_table.tex}
+    \end{footnotesize}
+}
 \caption{\label{tab:catalog}
-Details of the initial set of population models across four species.
-\textbf{NOTES: Column names are ID, name (or description maybe?),
-reference, CPU time, RAM}
+Initial set of demographic models and simple benchmarks.
```
Comment (agladstein) [inline popSimManu.tex]: "Initial set of demographic models included in the Catalog and simple benchmarks"

**Comment by agladstein** [comment]
Looks good!

**Review by andrewkern — APPROVED** [review]
this looks great. now a decision-- do we stick with referring to the models throughout the paper by their citations (as the group has been doing) or do we switch to this set of IDs?

**Comment by agladstein** [comment]
refer to them in text by ID and (citation)

File: popSimManu.tex (hunk 1 above)
Comment (jeromekelleher) [reply popSimManu.tex]: Done, thanks.

**Comment by jeromekelleher** [comment]
Great, thanks, going to merge to keep moving.

> this looks great. now a decision-- do we stick with referring to the models throughout the paper by their citations (as the group has been doing) or do we switch to this set of IDs?

Let's take this back to the original issue in #42.

**Pull request merged**